examples/Pyvista_drive_demo.ipynb

```diff
@@ -0,0 +1,688 @@
+    "\n",
+    "    # Panel layout\n",
+    "    layout = pn.Row(\n",
+    "        slider,\n",
+    "        time_step,\n",
+    "        styles={\n",
+    "            \"background\": \"black\",\n",
+    "            \"padding\": \"10px\",\n",
+    "            \"border-radius\": \"10px\",\n",
+    "            \"border\": \"1px solid white\",\n",
+    "        },\n",
+    "    )\n",
+    "\n",
+    "    display(layout)\n",
+    "    plot.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "62b40d4f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def drive_pv_streamlines(drive):\n",
+    "    \"\"\"\n",
+    "    pyvista streamline plot.\n",
+    "\n",
+    "    This function generates a plot of streamines. We use a\n",
+    "    panel slider change the time step and a panel number\n",
+    "    indicator to label the correspondingindex for the time\n",
+    "    step we are on.\n",
+    "\n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    drive:\n",
+    "\n",
+    "        Drive object that is used for analysis\n",
+    "    \"\"\"\n",
+    "    # Create a PyVista plotter\n",
+    "    plot = pv.Plotter()\n",
+    "\n",
+    "    def time_pv(value):\n",
+    "        drive[value].pyvista.streamlines(name=\"x\", plotter=plot)\n",
+    "        time_step.value = (drive.info[\"t\"] / drive.n) * (value + 1)\n",
+    "\n",
+    "    # Create a Panel slider\n",
+    "    slider = pn.widgets.IntSlider(\n",
+    "        name=\"Index\", value=0, start=0, end=drive.n - 1, step=1\n",
+    "    )\n",
+    "\n",
+    "    # Create a Panel FloatInput for the time step\n",
+    "    time_step = pn.indicators.Number(\n",
+    "        name=\"t:\",\n",
+    "        format=\"{value:.2g}\",\n",
+    "        title_size=\"10pt\",\n",
+    "        font_size=\"10pt\",\n",
+    "        default_color=\"white\",\n",
+    "    )\n",
+    "\n",
+    "    # Bind the slider to update function\n",
+    "    slider.param.watch(\n",
+    "        lambda event: time_pv(event.new), \"value\"\n",
+    "    )  # callback function where event.new is the newest slider value\n",
+    "\n",
+    "    # Initialize with first value\n",
+    "    time_pv(slider.value)\n",
+    "\n",
+    "    # Panel layout\n",
+    "    layout = pn.Row(\n",
+    "        slider,\n",
+    "        time_step,\n",
+    "        styles={\n",
+    "            \"background\": \"black\",\n",
+    "            \"padding\": \"10px\",\n",
+    "            \"border-radius\": \"10px\",\n",
+    "            \"border\": \"1px solid white\",\n",
+    "        },\n",
+    "    )\n",
+    "\n",
+    "    display(layout)\n",
+    "    plot.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07c56ab6",
+   "metadata": {},
+   "source": [
+    "## Code examples\n",
+    "\n",
+    "In the cells below we have an example of our `drive_pv_vector()` function and the holoviews equivalent, `drive.hv()`, to compare it with."
```

Comment:
- Change heading and text: something along the lines "the rest of the notebook demonstrates how to use the new plotting functions"
- Show all four functions
- For the notebook I would probably drop the comparison to `.hv`